featuretools/feature_base/feature_base.py

```diff
@@ -328,21 +328,23 @@ def unique_name(self):
 class IdentityFeature(FeatureBase):
     """Feature for entity that is equivalent to underlying variable"""
 
-    def __init__(self, variable):
+    def __init__(self, variable, name=None):
```

Comment:
If we are going to add name=None to every single child class of `FeatureBase`, then doesn't make sense to add it directly to the `FeatureBase` class?